_playwright-tests/UI/TemplateCRUD.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { test, expect } from '@playwright/test';
+import { navigateToRepositories, navigateToTemplates } from './helpers/navHelpers';
+import { closePopupsIfExist, getRowByNameOrUrl } from './helpers/helpers';
+import { deleteAllTemplates } from './helpers/deleteTemplates';
+import { deleteAllRepos } from './helpers/deleteRepositories';
+
+const templateNamePrefix = 'template_CRUD';
+const repoNamePrefix = 'custom_repo-template';
+
+export const randomName = () => (Math.random() + 1).toString(36).substring(2, 6);
+export const repoName = `${repoNamePrefix}-${randomName()}`;
+export const rank = () => Math.floor(Math.random() * 10 + 1).toString();
+export const randomNum = () =>
+  Math.floor(Math.random() * 10 + 1)
+    .toString()
+    .padStart(2, '0');
```

Comment:
I think you have done this in different test as well - maybe we can move this to helpers? (As separate commit)

Reply:
done that